subcommand/flags/flag_slice_value.go

```diff
@@ -0,0 +1,23 @@
+package flags
+
+import "strings"
+
+// Taken from https://github.com/hashicorp/consul/blob/master/command/flags/flag_slice_value.go
```

Comment:
Could you make all the links tied to a specific git sha? If this code is moved or deleted, then we'd lose this reference.

Reply:
good call, done!